types/src/verifier/mod.rs

```diff
@@ -3,10 +3,11 @@
 
 use filecoin_proofs_api::{post, PublicReplicaInfo};
 use filecoin_proofs_api::{ProverId, SectorId};
+use forest_shim::randomness::Randomness;
 use fvm_ipld_encoding::bytes_32;
 use fvm_shared::address::Address;
 use fvm_shared::commcid::cid_to_replica_commitment_v1;
-use fvm_shared::randomness::Randomness;
+use fvm_shared::randomness::Randomness as Randomness_v2;
```

Comment:
@sudo-shashank The goal is to *not* have the notion of `v2 and `v3` in Forest but to have it abstracted away via the shim. So in short, the shouldn't be any place in Forest (expect for the shim) that imports or uses explicitly `fvm_shared::randomness::Randomness`

Reply:
got it